relays/bin-substrate/src/cli/encode_call.rs

```diff
@@ -268,12 +268,8 @@ mod tests {
 
 		// then
 		assert_eq!(err.kind, structopt::clap::ErrorKind::ArgumentConflict);
-		assert_eq!(
-			err.info,
-			Some(vec![
-				"remark-size".to_string(),
-				"--remark-payload <remark-payload>".to_string()
-			])
-		);
+
+		let info = err.info.unwrap();
+		assert!(info.contains(&"remark-payload".to_string()) | info.contains(&"remark-size".to_string()))
```

Comment:
Should the `err.info` contain both, `remark-payload` and `remark-size`. So, it would be `&` instead of `|`.

Reply:
Actually I think it would be best to simply sort both vectors before comparing.

Reply:
In our case `err.info` will be either `Some(["remark-payload", "--remark-size <remark-size>"])` or `Some(["remark-size", "--remark-payload <remark-payload>"])`.

This means we can't check for both variants (e.g the first vector doesn't `contain` "remark-size"), and because of this sorting the vector won't help.